boards/ikea-tradfri/doc.txt

```diff
@@ -2,4 +2,169 @@
 @defgroup    boards_ikea-tradfri IKEA TRÅDFRI modules
 @ingroup     boards
 @brief       Support for the IKEA TRÅDFRI modules
+
+## Overview
+The IKEA TRÅDFRI is a small board found in different IKEA TRÅDFRI products.
+The board contains an EFR32 microcontroller and a 2MBit SPI NOR Flash.
+
+More information about the module can be found on this
+[GitHub page](https://github.com/basilfx/TRADFRI-Hacking).
+
+## Hardware
+
+### MCU
+| MCU             | EFR32MG1P132F256GM32                                                                         |
+|-----------------|----------------------------------------------------------------------------------------------|
+| Family          | ARM Cortex-M4F                                                                               |
+| Vendor          | Silicon Labs                                                                                 |
+| Vendor Family   | EFR32 Mighty Gecko 1P                                                                        |
+| RAM             | 31.0KB                                                                                       |
```

Comment:
The doc says 32KB. Is this a typo ?

```suggestion
| RAM             | 31.0KB                                                                                       |
```

Reply:
Technically, it's correct. But when you use the radio library (RAIL), it uses 1KiB of RAM.

This is the same as with the SLTB001a.

Reply:
Changed this into 32.0KB (1.0KB reserved). That's what the linker script uses.